src/Jackett.Common/Indexers/DigitalHive.cs

```diff
@@ -195,43 +198,45 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             return releases;
         }
 
-        private IEnumerable<ReleaseInfo> contentToReleaseInfos(TorznabQuery query, CQ dom)
+        private IEnumerable<ReleaseInfo> contentToReleaseInfos(TorznabQuery query, string content)
         {
+            var parser = new HtmlParser();
+            var dom = parser.ParseDocument(content);
             var releases = new List<ReleaseInfo>();
 
             // Doesn't handle pagination yet...
-            var rows = dom["div.panel-body > table.table > tbody > tr"];
+            var rows = dom.QuerySelectorAll("div.panel-body > table.table > tbody > tr");
             foreach (var row in rows)
             {
                 var release = new ReleaseInfo();
                 release.MinimumRatio = 1;
                 release.MinimumSeedTime = 259200;
 
-                var qRow = row.Cq();
-                release.Title = qRow.Find("td:nth-child(2) > a").First().Text().Trim();
+
+                release.Title = row.QuerySelector("td:nth-child(2) > a").TextContent.Trim();
 
                 if ((query.ImdbID == null || !TorznabCaps.SupportsImdbMovieSearch) && !query.MatchQueryStringAND(release.Title))
                     continue;
 
-                release.Guid = new Uri(SiteLink + qRow.Find("td:nth-child(2) > a").First().Attr("href"));
+                release.Guid = new Uri(SiteLink + row.QuerySelector("td:nth-child(2) > a").GetAttribute("href"));
                 release.Comments = release.Guid;
-                release.Link = new Uri(SiteLink + qRow.Find("td:nth-child(3) > a").First().Attr("href"));
-                var pubDateElement = qRow.Find("td:nth-child(2) > span").First();
-                pubDateElement.Find("a").Remove(); // remove snatchinfo links (added after completing a torrent)
-                var pubDate = pubDateElement.Text().Trim().Replace("Added: ", "");
+                release.Link = new Uri(SiteLink + row.QuerySelector("td:nth-child(3) > a").GetAttribute("href"));
+                var pubDateElement = row.QuerySelector("td:nth-child(2) > span");
+                pubDateElement.QuerySelector("a").Remove(); // remove snatchinfo links (added after completing a torrent)
```

Comment:
Are you sure it's removing only the first element? In other PRs you are removing all children => https://github.com/Jackett/Jackett/pull/7358/files#diff-3b7d74c8c28dd8eaf4f9364a5384f6e1R112-R113

Reply:
This will need to be changed, because for the most part, I did Find+Replace and fixed anything that errored out to most closely resemble the original csQuery code. `.Find(*)` was changed to `.QuerySelector(*)` and when that caused an error I changed it to `.QuerySelectorAll(*)` this line didn't cause an error, but introduced a bug instead.